### PR TITLE
Bake k8scloudconfig into Docker image

### PR DESCRIPTION
**Pull request by corest**
Towards https://github.com/giantswarm/giantswarm/issues/2816 (option 2 from here https://github.com/giantswarm/giantswarm/issues/2816#issuecomment-424759971)

Looks creepy as we bring all unused k8scloudconfig directories, but there is no other way to do this with `dep`, or I don't know any other way.

### CODE REVIEW

**Comment by corest** [comment]
If we're ok with this, I'll provide same PRs for other operators.

File: Gopkg.toml [hunk 1]
```diff
@@ -145,7 +145,10 @@ required = [
   name = "k8s.io/client-go"
   version = "kubernetes-1.10.4"
 
-
+[[prune.project]]
+  name = "github.com/giantswarm/k8scloudconfig"
+  unused-packages = false
```
Comment (kopiczko) [inline Gopkg.toml]: I wonder if we can drop this line, i.e. `unused-packages = false`.

Comment (corest) [reply Gopkg.toml]: Nope, I tried that. The only possible way to bypass that - add go file into each folder and import packages somewhere.

Comment (kopiczko) [reply Gopkg.toml]: That makes sense. It should be better when we have release-operator in place anyway.

**Pull request merged**